Mathlib/CategoryTheory/Monoidal/Mon_.lean

```diff
@@ -797,6 +800,44 @@ same on monoid objects as on objects. -/
 
 end Monoidal
 
+section BraidedCategory
+variable [BraidedCategory C] [BraidedCategory D] (F)
+
+open scoped Obj
+
+attribute [-simp] IsMonHom.one_hom_assoc in
+attribute [local simp← ] tensorHom_comp_tensorHom tensorHom_comp_tensorHom_assoc in
+attribute [simp] tensorμ_comp_μ_tensorHom_μ_comp_μ_assoc MonObj.tensorObj.one_def
+  MonObj.tensorObj.mul_def in
```

Comment:
Unless there has been a very recent change, should not this be `attribute [local simp]`?

Reply:
Whoops!